src/config/network/wicd.md

```diff
@@ -0,0 +1,13 @@
+# Wicd
+
+> Warning: Wicd sees no active development. The last release occurred in 2016,
+> and the program has not been ported to Python3. Use of Wicd is not
+> recommended.
+
+Wicd can be used to manage wired and wireless network interfaces, much like
+[NetworkManager](./networkmanager.md) or [ConnMan](./connman.md).
+
+Wicd does not require `dhcpcd` or `wpa_supplicant` services managed by runit.
```

Comment:
Would it be more accurate to say that Wicd conflicts with these services?

Reply:
yeah probably.